praktikum/poglavlja/09_strukture/index.rst

```diff
@@ -0,0 +1,161 @@
+tipa ``struct automobil`` mora navesti ``a.kubikaza`` ukoliko je potrebna vrednost promenljive, dok se za korišćenje adrese mora navesti ``&a.kubikaza``. Isto
+se odnosi i na ostala polja strukture, s tim da se sa stringom radi bez korišćenja ``&`` operatora za pristup adresi.
+
+Ispis programa:
+
+.. literalinclude:: primeri/automobil-funkcije-ispis.txt
+   :language: none
+
+Typedef
+-------
+
+Ključna reč ``typedef`` se koristi za preimenovanje tipa (pravljenje aliasa) za određeni tip podatka. U slučaju strukture, zgodna nam je za korišćenje
+kako bi pojednostavili sintaksu.
+Osnovna forma ovakve strukture je::
+
+  typedef struct ime_strukture
+  {
+    tip promenljiva_1;
+    tip promenljiva_2;
+    ...
+  } NOVO_IME;
+
+gde je novokreirani tip podatka ``NOVO_IME`` umesto ``struct ime_strukture``.
+
+Primer unosa automobila kroz funkciju
+-------------------------------------
+
+.. tabs::
+
+  .. tab:: Bez typedef
+    .. literalinclude:: primeri/automobil-funkcije.c
+       :linenos:
+  .. tab:: Korišćenjem typedef
+    .. literalinclude:: primeri/automobil-typedef.c
+       :linenos:
+
+U oba primeru iznad deklarisane su 2 promenljive. U primeru bez ``typedef``, promenljive su tipa ``struct automobil`` i zovu se
+``a1`` i ``a2``, dok su u primeru sa ``typedef`` promenljive takođe tipa ``struct automobil``, ali se umesto ovog tipa može koristiti alias
+``AUTO``. Prvo se poziva funkcija ``unos_automobila`` za ``a1`` kroz koju se unose vrednosti
+svih polja, pa se isto ponavlja i za ``a2``. Nakon toga se poziva funkcija ``ispis_automobila`` za ``a1``, pa za ``a2``. Funkciji
+``unos_automobila`` se struktura prosleđuje po pokazivaču, pa je pristup poljima strukture drugačiji nego u prvom primeru.
+Pristup strukturi prosleđenoj po pokazivaču je moguć na 2 načina:
+
+#. korišćenjem kombinacije ``*`` i ``.`` operatora
+#. korišćenjem ``->`` operatora
+
+Napomena:
+
+.. literalinclude:: objasnjenja/dereferenciranje.txt
+   :language: none
+
+Obe varijante primera daju isti ispis programa:
+
+.. literalinclude:: primeri/automobil-funkcije-ispis.txt
+   :language: none
+
+Primer rada sa nizovima struktura
+---------------------------------
+
+.. literalinclude:: primeri/automobili-niz.c
+   :linenos:
+
+U primeru iznad je definisana struktura ``struct automobil`` kojoj je dodeljen alias ``AUTO`` koji se u daljem radu programa koristi.
+Prvo se poziva funkcija ``unos`` kroz koju se unosi broj automobila, a zatim i vrednosti svih polja za svaki automobil. Kada se unesu
+svi automobili, poziva se funkcija ``ispis`` koja ispisuje podatke o svakom pojedinačnom automobilu.
+
+Ispis programa:
+
+.. literalinclude:: primeri/automobili-niz-ispis.txt
+   :language: none
+
+Učitavanje podataka u program
+-----------------------------
+
+Podatke smo do sada učitavali u program kroz standardi ulaz (terminal). Podaci se mogu programu proslediti i kroz drugi izvor,
```

Comment:
Greška u kucanju, "standardi".

Reply:
popravljeno

Reply:
Proveri da li si dobro komitovala, meni i dalje pokazuje "standardi".

Reply:
pogledaj samo poslednji komit, mislim da ne vidis jer je vec approved, deluje mi da je tamo sve ok, i to i ove recenice i primeri